<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TreapDemonstration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="dragMode">
         <enum>QGraphicsView::ScrollHandDrag</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Создание</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Добавить элементы</string>
     </property>
     <addaction name="from_file"/>
     <addaction name="from_string"/>
     <addaction name="generate"/>
    </widget>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Сохранить</string>
     </property>
     <widget class="QMenu" name="menu_3">
      <property name="title">
       <string>В Файл</string>
      </property>
      <addaction name="NLR"/>
      <addaction name="LNR"/>
      <addaction name="LRN"/>
     </widget>
     <addaction name="menu_3"/>
    </widget>
    <addaction name="menu_4"/>
    <addaction name="menu_5"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Редактирование</string>
    </property>
    <addaction name="add_item"/>
    <addaction name="delete_item"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="from_file">
   <property name="text">
    <string>Из Файла</string>
   </property>
  </action>
  <action name="from_string">
   <property name="text">
    <string>Из Строки</string>
   </property>
  </action>
  <action name="generate">
   <property name="text">
    <string>Сгенерировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="add_item">
   <property name="text">
    <string>Добавить Элемент</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="delete_item">
   <property name="text">
    <string>Удалить Элемент</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="NLR">
   <property name="text">
    <string>Прямой обход (NLR)</string>
   </property>
  </action>
  <action name="LNR">
   <property name="text">
    <string>Центрированный обход (LNR)</string>
   </property>
  </action>
  <action name="LRN">
   <property name="text">
    <string>Обратный обход (LRN)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
